src/video/H264Encoder.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <string>
+#include <vector>
+
+#include <H264Encoder/encoder.hpp>
+#include <opencv2/core/core.hpp>
+
+namespace video {
+class H264Encoder {
+private:
+	std::unique_ptr<h264encoder::Encoder> encoder;
+	int fps;
+
+public:
+	H264Encoder(int fps);
```

Comment:
Can we get doxy comments for the class and the constructor? And also we prefer to put the public section above the private, so can you switch the order?